test/specs/practise/PracticeCode.spec.js

```diff
@@ -6,118 +6,110 @@ import { EnvNames, NetworkNames, Sdk } from "etherspot";
 import { BigNumber } from "ethers";
```

Comment:
What is the purpose of the practice file and should it be skipped?

Reply:
The practice file is for practice purposes, So I initially write the code in the practice file and then implement it for each network as smoke and regression test cases.